README.md

```diff
@@ -160,54 +147,6 @@ private static void MqttConnectionOpened(object sender, ConnectionOpenedEventArg
 }
 ```
 
```

Comment:
The usage and example are quite all right here. No need to remove. It's independent from having the samples here or at the other repo.